lib/prompts/projectAddPrompt.ts

```diff
@@ -47,4 +57,13 @@ export async function projectAddPrompt(
       },
     },
   ]);
+
+  if (providedTypeIsValid) {
```

Comment:
Doing this to match the pattern in createProjectPrompt. It makes it so this prompt will always return a value for `componentData`